calicoctl.py

```diff
@@ -783,6 +759,47 @@ def ip_pool_show(version):
     print x
 
 
+def restart_docker_with_alternative_unix_socket():
+    # Update docker to use a different unix socket, so powerstrip can run
+    # its proxy on the "normal" one. This provides simple access for
+    # existing tools to the powerstrip proxy.
+
+    # Set the docker daemon to listen on the docker.real.sock by updating
+    # the config, clearing old sockets and restarting.
+    socket_config_exists = \
+        DOCKER_OPTIONS in open(DOCKER_DEFAULT_FILENAME).read()
```

Comment:
Should also put this in a `try:` block with a helpful error message if `DOCKER_DEFAULT_FILENAME` doesn't exist.